extensions/wikia/NjordPrototype/maintenance/RemoveNjordTag.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Maintenance script for removing NJORD (MOM)
+ * Usage: $ SERVER_ID=1 php /usr/wikia/source/wiki/extensions/wikia/NjordPrototype/maintenance/RemoveNjordTag.php
+ */
+
+require_once( __DIR__ . '/../../../../maintenance/Maintenance.php' );
+
+class RemoveNjordTag extends Maintenance {
+	/**
+	 * Duplicated with NjordUsage script;
+	 * Not refactoring as a part of feature sunset process
+	 */
+	const NJORD_ARTICLE_PROP_TITLE = 'TITLE';
+	const NJORD_ARTICLE_PROP_DESCR = 'DESCRIPTION';
+	const NJORD_ARTICLE_PROP_IMAGE = 'IMAGE';
+	const NJORD_ARTICLE_CROP_POSITION = 'CROP_POSITION';
+	const WIKI_CREATION_DATE = 'WIKI_CREATION_DATE';
+
+	private $dryRun = true;
+
+	function __construct() {
+		$this->addOption( 'process', 'Do the real operation. As the operation is destructive, without this flag it will only perform dry-run' );
+		parent::__construct();
+	}
+
+	public function execute() {
+		global $wgCityId, $wgUser;
+
+		// attribute automated edits to FandomBot
+		$wgUser = User::newFromName( 'FandomBot' );
+
+		$this->dryRun = !$this->hasOption( 'process' );
+
+		if ( $this->dryRun ) {
+			$this->output( 'Dry run for ' . $wgCityId . PHP_EOL );
+		} else {
+			$this->output( PHP_EOL );
+			$this->output( '!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!' . PHP_EOL );
+			$this->output( 'REAL DESTRUCTIVE OPERATION: Removal for ' . $wgCityId . PHP_EOL );
+			$this->output( '!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!' . PHP_EOL );
+			$this->output( PHP_EOL );
+		}
+
+		$this->removeNjordTag();
+		$this->removeNjordPagePropsData();
+		$this->output( PHP_EOL );
+
+		$this->output( 'Done for ' . $wgCityId . PHP_EOL );
+	}
+
+	private function removeNjordTag() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this function name "removeNjordTag", since a "__construct" method has already been defined in this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1603)